proto/kvrpcpb.proto

```diff
@@ -791,48 +791,54 @@ message Context {
 // The API version the server and the client is using.
 // See more details in https://github.com/tikv/rfcs/blob/master/text/0069-api-v2.md.
 enum APIVersion {
-    // Mainly for TxnKV and not safe to use RawKV along with TxnKV.
-    //
-    // V1 server only accepts V1 requests. Except that the V1 raw requests with TTL
-    // will be rejected.
+    // `V1` is mainly for TiDB & TxnKV, and is not safe to use RawKV along with the others.
+    // V1 server only accepts V1 requests. V1 raw requests with TTL will be rejected.
     V1 = 0;
-    // Only RawKV is available, and then 8 bytes representing the unix timestamp in
-    // seconds for expiring time will be append to the value of all RawKV kv pairs.
     //
+    // `V1TTL` is only available to RawKV, and 8 bytes representing the unix timestamp in
+    // seconds for expiring time will be append to the value of all RawKV entries. For example:
     // ------------------------------------------------------------
     // | User value     | Expire Ts                               |
     // ------------------------------------------------------------
     // | 0x12 0x34 0x56 | 0x00 0x00 0x00 0x00 0x00 0x00 0xff 0xff |
     // ------------------------------------------------------------
-    //
     // V1TTL server only accepts V1 raw requests.
     // V1 client should not use `V1TTL` in request. V1 client should always send `V1`.
     V1TTL = 1;
-    // TxnKV keys start with `x{keyspace id}`, `m`, or `t`.
-    //
-    // RawKV keys must be in `default` CF and all start with `r{keyspace id}` prefix,
-    // where the keyspace id is in varint format (little endian), whose bytes expect
-    // the last one always sets the most significant bit to 1.
-    //
-    // The last byte in the raw value must be a meta flag. For example:
     //
-    // --------------------------------------
-    // | User value     | Meta flags        |
-    // --------------------------------------
-    // | 0x12 0x34 0x56 | 0x00 (0b00000000) |
-    // --------------------------------------
+    // `V2` use new encoding for RawKV & TxnKV to support more features.
     //
-    // As shown in the example below, the least significant bit of the meta flag
-    // indicates whether the value contains 8 bytes expire ts at the very left to the
-    // meta flags.
+    // Key Encoding:
+    //  TiDB: start with `m` or `t`, the same as `V1`.
```

Comment:
Because `m` and `t` are added in the logic of TiDB,
We can say that : TiDB uses the v1 client, and user key starts with `m` and `t` ?